### ISSUE TITLE
AttributeError: 'EncodedMethod' object has no attribute 'get_instructions_idx'

### ISSUE THREAD

**Issue by dvh-96** [androguard/androguard#793]
I'm trying to analyze the provided TestActivity.apk. Currently I am working myself through the documentation (https://androguard.readthedocs.io/en/latest/intro/instructions.html#getting-disassembled-instructions). 

However, when executing the following bit of code:

```
for method in dx.get_methods():
    if method.is_external():
        continue
    m = method.get_method()
    for idx, ins in m.get_instructions_idx():
        print(idx, ins.get_op_value(), ins.get_name(), ins.get_output())
```

I got an error, instead of the output: 
`AttributeError: 'EncodedMethod' object has no attribute 'get_instructions_idx'`

Am I missing something?
Thanks in advance!

### System Information

- Androguard Version: master (v3.3.5)
- Python Version: 3.6 (anaconda)
- Operating System: Windows 10

**Comment by krnick** [comment]
Hi @dvh-96,

The `get_instructions_idx` method is only available in version 3.4, but currently version 3.3.5 provided on PyPi does not have this feature.

You can refer to version master in [dvm.py](https://github.com/androguard/androguard/blob/22849b6965835b611f5fd6615de9301f5f47318e/androguard/core/bytecodes/dvm.py#L3128-L3142)

So far, you can install androguard directly via pip install from GitHub to use this feature.

The command as below:

```
pip3 install -e git+https://github.com/androguard/androguard.git#egg=androguard
```

Same issue you could refer #755

**Comment by dvh-96** [comment]
Thanks! @krnick

That indeed did the job. I didn't know that it was only available in version 3.4
